.circleci/config.yml

```diff
@@ -805,6 +813,7 @@ workflows:
       - test_build_py36: *requires_pre_test
       - test_build_py35: *requires_pre_test
       - test_build_py27: *requires_pre_test
+      - build_wheels: *requires_pre_test
```

Comment:
great!

Reply:
I wonder if we can remove the `test_build_py*` ones too? if we are building the wheels this is:

- build sdist
- build wheel for every version

Reply:
I actually started doing that and stopped.
The `test_build_py` jobs use `DDTRACE_BUILD_RAISE` which seems valuable.